client/my-sites/google-my-business/stats/index.js

```diff
@@ -66,36 +66,58 @@ class GoogleMyBusinessStats extends Component {
 		} );
 	};
 
-	renderViewsTooltipForDatanum = ( datanum, dataSeries ) => {
-		const { value: valueCount, date } = datanum;
-		if ( dataSeries.length > 20 ) {
+	renderViewsTooltipForDatanum = ( datanum, interval ) => {
+		const { value: viewCount, date } = datanum;
+		if ( interval === 'quarter' ) {
+			return this.props.translate(
+				'%(value)d view on the week of %(monday)s',
+				'%(value)d views on the week of %(monday)s',
+				{
+					count: viewCount,
+					args: {
+						value: viewCount,
+						monday: moment( date ).format( 'LL' ),
+					},
+				}
+			);
+		} else if ( interval === 'week' ) {
```

Comment:
Is there a reason why `month` doesn't get the new style of tooltip?

Reply:
Yeah, let's set the new style for month interval as well 👍